easybuild/easyblocks/q/quantumespresso.py

```diff
@@ -49,7 +49,6 @@ class EB_QuantumESPRESSO(ConfigureMake):
     def extra_options():
         """Custom easyconfig parameters for Quantum ESPRESSO."""
         extra_vars = {
-            'hybrid': [False, "Enable hybrid build (with OpenMP)", CUSTOM],
```

Comment:
I think we need to retain this (for now), otherwise people who use `hybrid = True` now without having `openmp` enabled in `toolchainopts` will run into a different build?